mypy.ini

```diff
@@ -10,7 +10,7 @@ disallow_untyped_calls = True
 disallow_untyped_defs = True
 disallow_incomplete_defs = True
 check_untyped_defs = True
-disallow_untyped_decorators = True
+disallow_untyped_decorators = False
```

Comment:
@altendky I couldn't figure out a way around this. It looks like either an issue in mypy or pytest.

Without this I get:

```
tests/core/util/test_db_wrapper.py:51: error: Untyped decorator makes function "test_concurrent_writers" untyped  [misc]
tests/core/util/test_db_wrapper.py:52: error: Untyped decorator makes function "test_concurrent_writers" untyped  [misc]
tests/core/util/test_db_wrapper.py:84: error: Untyped decorator makes function "test_writers_nests" untyped  [misc]
tests/core/util/test_db_wrapper.py:103: error: Untyped decorator makes function "test_partial_failure" untyped  [misc]
tests/core/util/test_db_wrapper.py:126: error: Untyped decorator makes function "test_readers_nests" untyped  [misc]
tests/core/util/test_db_wrapper.py:142: error: Untyped decorator makes function "test_readers_nests_writer" untyped  [misc]
tests/core/util/test_db_wrapper.py:158: error: Untyped decorator makes function "test_concurrent_readers" untyped  [misc]
tests/core/util/test_db_wrapper.py:159: error: Untyped decorator makes function "test_concurrent_readers" untyped  [misc]
tests/core/util/test_db_wrapper.py:188: error: Untyped decorator makes function "test_mixed_readers_writers" untyped  [misc]
tests/core/util/test_db_wrapper.py:189: error: Untyped decorator makes function "test_mixed_readers_writers" untyped  [misc]
Found 10 errors in 1 file (checked 532 source files)
```